styleguide/components/31-molecules/social-list/_social-list.scss

```diff
@@ -1,18 +1,19 @@
 $social-media-link-width: 1.6rem;
 
 .social-list {
-  width: 100%;
+  display: inline-block;
+  width: auto;
+  max-width: 100%;
```

Comment:
Why is this needed?
Seems silly, but I'm sure you had your reason.

Reply:
Totally unnecessary; removed it 👍